### PR TITLE
rename rpcs

### PR DESCRIPTION
**Pull request by pifragile**
Introduce `encointer_` prefix for RPC method names
Part of #163

### CODE REVIEW

File: bazaar/rpc/src/lib.rs [hunk 1]
```diff
@@ -33,19 +33,19 @@ pub trait BazaarApi<BlockHash, AccountId>
 where
 	AccountId: 'static + Encode + Decode + Send + Sync,
 {
-	#[rpc(name = "bazaar_getBusinesses")]
+	#[rpc(name = "encointer_bazaar_getBusinesses")]
```
Comment (clangenb) [inline bazaar/rpc/src/lib.rs:36]: The rpc macro does not allow for double underscores, we need to do it like:

```
encointer_bazaarGetBusinsses
```

**Comment by clangenb** [comment]
Now that the other PR has already been merged, you need to do the same for the aggregated account data. :)

**Comment by brenzi** [comment]
please always describe a PR and mention the issue it closes

**Comment by pifragile** [comment]
ok, i wasn't sure because it does not acutally close the issue (if we consider the refactoring of the rpc into a separate crate as part of the issue.) But i will mention it anyway

**Review by clangenb — APPROVED** [review]
LGTM, matches the result of our discussion. What is your take on this @brenzi?

**Comment by brenzi** [comment]
Good to see that this does not touch the runtime at all.

**Pull request merged**